tests/test_main.py

```
@@ -6,16 +6,53 @@
 from datetime import datetime
 import os
 import pandas as pd
+from tests.test_constants import PROBLEM_TYPE
```

Ruff (0.8.2)

`tests.test_constants.PROBLEM_TYPE` imported but unused

Remove unused import: `tests.test_constants.PROBLEM_TYPE`

(F401)